.cd-builders/windows/builder/Makefile

```diff
@@ -1,4 +1,9 @@
 check:
 	ansible windows -i hosts -m win_ping
 provision:
-	ansible-playbook -i hosts playbook_build_env.yml
+	ansible-playbook -i hosts builder/playbook_build_env.yml
+create-win-ami:
+	packer build packer-agent6-winrunner.json
```

Comment:
it would be great to rebuild the AMI once in a while aswell (automated) to avoid having it stale again.. maybe you can put a tech debt story on the backlog tagged with intermission to make this happen?

Reply:
Addressed as a  STAC-11659